apps/backend/.env.test

```
@@ -33,6 +33,7 @@ AWS_LOCATION_PLACE_INDEX_NAME=open-jii-places-test
 AWS_COGNITO_IDENTITY_POOL_ID=moon-base-1:test-pool-id
 AWS_COGNITO_DEVELOPER_PROVIDER_NAME=login.openjii.com
 AWS_IOT_POLICY_NAMES=test-iot-policy,test-iot-script-policy
+AWS_IOT_JOBS_POLICY_NAME=open_jii_local_iot_policy_jobs
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AWS_IOT_JOBS_POLICY_NAME key should go before the AWS_IOT_POLICY_NAMES key

(UnorderedKey)